### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by moise-mulungu**
On this milestone, I:
- Added four URLs(`users`, `posts`, `comments` and `likes`).
- Created a route for each URL.
- Created `index` and `show` actions for `posts` and `users` controllers.
- Created view files for `index` and `show` actions in `posts` and `users` controllers.
- Added placeholder for each view file.
- Added code to handle the four required URLs.
- Updated the README file.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+    resources :likes
+    resources :comments
+  end
+  resources :posts do
+    resources :likes
+    resources :comments
+  end
+  resources :likes
+  resources :comments
   # Defines the root path route ("/")
```
Comment (DammyShittu) [inline config/routes.rb]: - [ ] At this point we need to expose ONLY the needed routes and those are the users' index and show routes; and the posts' index and show routes. The rest are not needed at this point. Kindly refactor your routes file accordingly. In this file we should only have these routes:

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @moise-mulungu,

✔️ Controller actions and views for users and posts

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (moise-mulungu) [reply config/routes.rb]: Thank you so much @DammyShittu

**Review by ShadyShawkat — APPROVED** [review]
## Hi @moise-mulungu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**